### PR TITLE
Implement color component

### PR DESCRIPTION
**Pull request by priyankaarkade**
- Created color component to check out what are colors are there for the product.
- Added css style for color component.
- Made changes in css product detail page to make column side by side.
- Let me know if i need to make any change or to refactor code.

### CODE REVIEW

File: src/containers/ProductColorSelector/index.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    width: 100%;
+    padding: 5px;
+}
+
+.product_color_selector__blue {
+    background-color: rgb(22, 22, 233);
+    width: 100%;
+    padding: 5px;
+}
+
```
Comment (kyle-arkade) [inline src/containers/ProductColorSelector/index.css]: newline

File: src/containers/ProductColorSelector/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          
+          if(!(variantColour in colorsAndImages)) {
+            let objColorAndImage = {};
+
+            objColorAndImage.colour = variantColour;
+            objColorAndImage.image_src = variant.image.src;
+
+            colorsAndImages[variantColour] = objColorAndImage;
+          }
+        });
```
Comment (kyle-arkade) [inline src/containers/ProductColorSelector/index.js]: newline

File: src/containers/ProductColorSelector/index.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                <div key={key} className="product_color_selector__hover">
+                    <div className="product_color_selector__image">
+                        <img src={object.image_src} alt="product_color_image" />
+                    </div>
+                    <div className="product_color_selector__color_value">
+                        <div className={"product_color_selector__" + object.colour}></div>
+                    </div>
+                </div>         
+            )   
+        })
```
Comment (kyle-arkade) [inline src/containers/ProductColorSelector/index.js]: ;

File: src/containers/ProductColorSelector/index.css (hunk 1 above)
Comment (priyankaarkade) [reply src/containers/ProductColorSelector/index.css]: resolved newline

File: src/containers/ProductColorSelector/index.js (hunk 2 above)
Comment (priyankaarkade) [reply src/containers/ProductColorSelector/index.js]: resolved newline

File: src/containers/ProductColorSelector/index.js (hunk 3 above)
Comment (priyankaarkade) [reply src/containers/ProductColorSelector/index.js]: resolved ;)

**Pull request merged**